modules/apps/object/object-test/src/testFunctional/tests/ObjectPersonalData.testcase

```diff
@@ -22,6 +22,273 @@ definition {
 		}
 	}
 
+	@description = "LPS-186468 - Verify that is possible to anonymize object entries"
```

Comment:
`Verify the object entries can be anonymized`

Same thing here, the ticket ID should be the story ticket.